### PR TITLE
Correction to logic_ternary type check

### PR DESCRIPTION
**Pull request by AnmAtAnm**

### CODE REVIEW

File: blocks/logic.js [hunk 1]
```diff
@@ -231,7 +231,7 @@ Blockly.defineBlocksWithJsonArray([  // BEGIN JSON EXTRACT
         "name": "ELSE"
       }
     ],
-    "output": true,
+    "output": null,
```
Comment (picklesrus) [inline blocks/logic.js]: I don't follow this part of the change. Can you explain some more? If I'm looking at the right thing, it looks like before this was converted to json we did a setOutput(true).

Comment (rachel-fenichel) [inline blocks/logic.js]: https://developers.google.com/blockly/guides/create-custom-blocks/define-blocks#block_output

Comment (picklesrus) [inline blocks/logic.js]: Ah, I see. Subtle. Thanks.

**Comment by picklesrus** [comment]
LGTM

File: blocks/logic.js (hunk 1 above)
Comment (AnmAtAnm) [inline blocks/logic.js]: Yeah.  This passed the logic blocks generator tests (so I didn't notice it earlier), but threw in an error in the texts blocks generator tests.

**Pull request merged**